packages/react-data-grid/src/common/types.ts

```diff
@@ -7,11 +7,11 @@ export type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
 
 export type SelectedRow<TRow> = TRow & { isSelected: boolean };
 
-export interface Column<TRow, TValue = unknown, TDependentValue = unknown> {
+interface ColumnValue<TRow, TDependentValue = any, TField extends keyof TRow = keyof TRow> {
   /** The name of the column. By default it will be displayed in the header cell */
   name: string;
   /** A unique key to distinguish each column */
-  key: keyof TRow;
+  key: TField;
```

Comment:
This makes value strongly typed

![image](https://user-images.githubusercontent.com/1452717/62506983-4e821180-b7c8-11e9-95e1-7754b59ff689.png)